yarn.lock

```diff
@@ -2,6 +2,11 @@
 # yarn lockfile v1
 
 
+"@alloc/quick-lru@^5.2.0":
```

Comment:
In general, what do you think about not including yarn.lock in PRs since it easily creates conflicts and instead have a separate PR that includes just the update?